configurations/actions-dev/resizeImage.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "resizeImage",
+  "displayName": "Resize Image",
+  "description": "Resizes an image to a specified width and height.",
+  "helpMessage": "",
+  "icon": "testIcon.svg",
+  "path": "/execute/resizeImage",
+  "parameters": {
+    "selectionfields": [
+      {
+        "name": "images",
```

Comment:
Why is the name of the selection field images? Would smth like cutout more suitable?